src/main.rs

```diff
@@ -8,7 +8,7 @@ use libp2p::identity::ed25519;
 use libp2p::mplex::MplexConfig;
 use libp2p::noise::{NoiseConfig, X25519Spec};
 use libp2p::ping::{Ping, PingConfig, PingEvent};
-use libp2p::rendezvous::{Config, Event as RendezvousEvent, Rendezvous};
+use libp2p::rendezvous::server::Config;
```

Comment:
Could get rid of this import if we refer to it via `rendezouvs::server::Config`.